Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift

```diff
@@ -66,12 +81,47 @@ extension TabTrayV2ViewController: UITableViewDataSource {
             viewModel.removeTab(forIndex: indexPath)
         }
     }
+    
+    @objc func didTapToolbarAddTab(_ sender: UIButton) {
+        viewModel.addTab()
+        dismissTabTray()
+    }
+    
+    @objc func didTapToolbarDelete(_ sender: UIButton) {
+        let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        controller.addAction(UIAlertAction(title: Strings.AppMenuCloseAllTabsTitleString, style: .default, handler: { _ in self.viewModel.closeTabsForCurrentTray() }), accessibilityIdentifier: "TabTrayController.deleteButton.closeAll")
+        controller.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment: "Label for Cancel button"), style: .cancel, handler: nil), accessibilityIdentifier: "TabTrayController.deleteButton.cancel")
```

Comment:
nit: I am guessing this string is temporary and will be moved to Strings file

Reply:
Ya I should change that. This block is copied from the old implementation.